core/shared/src/main/scala/sttp/client/RequestT.scala

```diff
@@ -320,6 +320,11 @@ class SpecifyAuthScheme[U[_], T, +S](hn: String, rt: RequestT[U, T, S]) {
 
   def bearer(token: String): RequestT[U, T, S] =
     rt.header(hn, s"Bearer $token")
+
+  def digest(user: String, password: String): RequestT[U, T, S] = {
+    import DigestAuthenticationBackend._
+    rt.digestAuth(user, password)
```

Comment:
shouldn't this take into account `hn` - the header name? (whether it's proxy or not)

Reply:
I don't think so, it just sets a tag on request which is later read by `DigestAuthenticatorBackend`

Reply:
but shouldn't `request.auth.digest` work differently from `request.proxyAuth.digest`?

Reply:
The proxy path works a little bit different than normal path, but it is all in `DigestAuthenticationBackend`